cmd/ocm/create/cluster/cmd.go

```diff
@@ -435,7 +504,12 @@ func preRun(cmd *cobra.Command, argv []string) error {
 		args.clusterWideProxy.Enabled = true
 	}
 
-	err = promptCCS(fs)
+	// If marketplace-gcp subscription type is used, ccs should by default be true
+	if isGcpMarketplaceSubscriptionType {
+		fmt.Println("setting ccs to 'true'")
+		args.ccs.Enabled = true
+	}
+	err = promptCCS(fs, args.ccs.Enabled == true)
```

Comment:
promptCCS(fs, args.ccs.Enabled == true) can go in a else block here, no ? This way we could avoid the extra parameter.

Reply:
We still need to call promtCCS as "service account file" prompt logic is present over there. And that is based on whether args.ccs.Enables is true or false

Reply:
okay... I was not expecting that.

Should we replace "args.ccs.Enabled == true" with  args.ccs.Enabled ?

Reply:
Simplified